app.js

```diff
@@ -1,9 +1,81 @@
 const express = require('express');
 const app = express();
-const createMyCustomApi = require('./routes');
+const fs = require('fs');
+
+//const createMyCustomApi = require('./routes');
```

Comment:
Is it needed? Should it be uncommented or deleted?
Isn't the part of the task is to keep routes in a separate file?

Reply:
I think keeping routes in a separate file is more consistent with the task description.

Reply:
Good. My main point was - there should be no commented out code in our PRs.